build/root/WORKSPACE

```diff
@@ -24,10 +23,7 @@ load("@io_k8s_repo_infra//:repos.bzl", repo_infra_configure = "configure", repo_
 #            'override_go_version': used to specify an alternate go version provided
 #                                   by kubernetes/repo-infra
 repo_infra_configure(
-    # TODO(go1.15): Uncomment once we've published a repo-infra release that supports go1.15
-    #go_version = "1.15.0",
-    # TODO(go1.15): Comment out once we've published a repo-infra release that supports go1.15
-    override_go_version = "1.15.0-rc.2",
+    go_version = "1.15.0",
```

Comment:
> unknown Go version: 1.15.0

should this be "1.15"?

Reply:
this missing ".0" really bugs me

Reply:
yeah :|
it's `1.15`

Reply:
It really bugs me too.

Reply:
w00t! i caught something in my review.... now make me an approver :)